### ISSUE TITLE
Add "delete cookies" on the browser of a specific extension 

### ISSUE THREAD

**Issue by Palloxin** [tachiyomiorg/tachiyomi#6917]
### Describe your suggested feature

![IMG_20220413_124234](https://user-images.githubusercontent.com/75091899/163163932-bd44f7a2-c914-47e5-85db-2501eebd0f8e.jpg)
here.
The feature is about a button that deletes the cookies of the specific website related to the extensions, so no other websites.
let me explain why. On extensions with cloudifre protection happens that the opening of a chapter is blocked by cloudifire, but if i use webview the cloudfire "trigger" (wait 5 seconds until... that) doesn't occour, and despite me opening a chapter directly on the webview, the "trigger" doesn't occour. i usually manage to trigger it after various attempts but it is very annoying every time. the button to delete cookies on this extension would improve greatly the experience.

https://user-images.githubusercontent.com/75091899/163164940-0053e043-5608-4295-b742-f459f2099146.mp4

This is what i am talking about.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
It already exists in the extension details view.

**Comment by CrepeTF** [comment]
So you want to make a change to the whole app, to make it easier to use a single extension when there already exist over 1000 other extensions instead of migrating away because.....?

**Comment by Palloxin** [comment]
it is not the whole app, it is just a button that already exist, but "nearer". + it doesn't only happens on that extension :/

**Comment by arkon** [comment]
I'll consider putting it in the WebView menu.

**Comment by Palloxin** [comment]
Ok then
